public/refs/paths/manage_global_form_template_{id}_create.json

```
@@ -0,0 +1,79 @@
+{
+  "post": {
+    "summary": "Create Form Template from Global Form Template",
+    "operationId": "postGlobalFormTemplateCreate",
```

lets have it as nice names - those are used in RTKQ hooks

```suggestion
    "operationId": "createFormTemplateFromGlobalFormTemplate",
```